erizo/src/erizo/DtlsTransport.cpp

```diff
@@ -288,33 +286,32 @@ std::string DtlsTransport::getMyFingerprint() {
 }
 
 void DtlsTransport::updateIceState(IceState state, NiceConnection *conn) {
-  ELOG_DEBUG( "%s - New NICE state state: %d - Media: %d - is Bundle: %d", transport_name.c_str(), state, mediaType, bundle_);
+  ELOG_DEBUG( "%s, message:NiceState, transport_name: %s, state: %d, bundle: %d", toLog(), transport_name.c_str(), state, bundle_);
   if (state == NICE_INITIAL && this->getTransportState() != TRANSPORT_STARTED) {
     updateTransportState(TRANSPORT_STARTED);
   }
   else if (state == NICE_CANDIDATES_RECEIVED && this->getTransportState() != TRANSPORT_GATHERED) {
     updateTransportState(TRANSPORT_GATHERED);
   }
   else if(state == NICE_FAILED){
-    ELOG_DEBUG("Nice Failed, no more reading packets");
+    ELOG_DEBUG("%s, message: Nice Failed", toLog());
     running_ = false;
     updateTransportState(TRANSPORT_FAILED);
   }
   else if (state == NICE_READY) {
-    ELOG_INFO("%s - Nice ready", transport_name.c_str());
     if (!isServer_ && dtlsRtp && !dtlsRtp->started) {
-      ELOG_INFO("%s - DTLSRTP Start", transport_name.c_str());
+      ELOG_INFO("%s, message: DTLSRTP Start, transport_name: %s", toLog(), transport_name.c_str());
       dtlsRtp->start();
     }
     if (!isServer_ && dtlsRtcp != NULL && (!dtlsRtcp->started || rtcpResender->getStatus() < 0)) {
-      ELOG_DEBUG("%s - DTLSRTCP Start", transport_name.c_str());
+      ELOG_DEBUG("%s, message: DTLSRTCP Start, transport_name: %s", toLog(), transport_name.c_str());
       dtlsRtcp->start();
     }
   }
 }
 
 void DtlsTransport::processLocalSdp(SdpInfo *localSdp_) {
-  ELOG_DEBUG( "Processing Local SDP in DTLS Transport" );
+  ELOG_DEBUG("%s, message: processing local sdp, transport_name: %s", toLog(), transport_name.c_str());
```

Comment:
`ELOG_DEBUG("%s, message: processing local sdp, transportName: %s", toLog(), transport_name.c_str());`

There are other transport_name in this file. Check the others.